test/module/irohad/model/model_mocks.hpp

```diff
@@ -22,11 +22,22 @@
 #include "ametsuchi/wsv_command.hpp"
 #include "ametsuchi/wsv_query.hpp"
 #include "model/command.hpp"
+#include "model/model_crypto_provider.hpp"
 #include "model/query_execution.hpp"
 
 namespace iroha {
   namespace model {
 
+    class MockCryptoProvider : public ModelCryptoProvider {
```

Comment:
ModelCryptoProvider is deprecated class and now it is used only in CLI. I don't know why this mock could be needed for.
I think changes in this file should be reverted.